src/api/app/models/bs_request.rb

```diff
@@ -189,7 +192,7 @@ def self.new_from_xml(xml)
 
       request.commenter = state.delete('who')
       unless request.commenter
-        raise 'no one logged in and no user in request' unless User.current
+        raise SaveError, 'no one logged in and no user in request' unless User.current
```

Comment:
Same here. A [validation on the presence](http://guides.rubyonrails.org/active_record_validations.html#presence).